### ISSUE TITLE
Atom tab landing side confusion

### ISSUE THREAD

**Issue by elliotcm** [atom/atom#15790]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

Atom tab dragging cannot decide which side of a tab to land on if cursor placed in the middle.

### Steps to Reproduce

Safe mode:
![atom-flappy-tab](https://user-images.githubusercontent.com/109225/31077356-23b4c51a-a776-11e7-8115-a22cdb358e95.gif)

**Expected behavior:** Tab landing indicator chooses left or right

**Actual behavior:** Tab landing indicator flips rapidly between left and right

**Reproduces how often:** 100% of the time

### Versions

```
Atom    : 1.20.1
Electron: 1.6.9
Chrome  : 56.0.2924.87
Node    : 7.4.0
```
```
apm  1.18.4
npm  3.10.10
node 6.9.5 x64
python 2.7.13
git 2.14.1
```

OSX 10.12.6 (16G29)

**Comment by rsese** [comment]
Thanks for the report!  I can reproduce on macOS 10.12.6 - I was going to say this should probably be moved to https://github.com/atom/tabs but I noticed I could only reproduce with the Atom Light and Dark UI themes, I can't reproduce with the One UI themes.  I also tried a couple of other UI themes (Apex and Material) but wasn't able to reproduce with those either so maybe this is an issue with the Atom themes?  I'll re-open this in https://github.com/atom/atom-light-ui/ and https://github.com/atom/atom-dark-ui/ today unless you get to it first @elliotcm :v:
